astroaccelerate.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@DEST_DIR@
+libdir=${prefix}/lib
+includedir=${prefix}/include
```

Comment:
See https://github.com/jtojnar/cmake-snips/

This is a bit complicated to do correctly with cmake -- I strongly advise to use that snippet to construct paths.

Your meson.build contribution could just use
```meson
pkgconfig = import('pkgconfig')
pkgconfig.generate(
    astroac_lib,
    description: 'AstroAccelerate is a many-core accelerated software package for processing time-domain radio-astronomy data.',
)
```

and it should work out of the box.

Reply:
Thanks, this is indeed much nicer than cmake!